erpnext/patches/v13_0/updates_for_multi_currency_payroll.py

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) 2019, Frappe and Contributors
+# License: GNU General Public License v3. See license.txt
+
+import frappe, erpnext
+
+def execute():
+	company = erpnext.get_default_company()
+	company_currency = erpnext.get_company_currency(company)
+	default_payroll_payable_account = frappe.db.get_value('Company', erpnext.get_company_currency(company), 'default_payroll_payable_account')
+	frappe.reload_doc('accounts', 'doctype', 'salary_component_account')
+	frappe.reload_doc('hr', 'doctype', 'employee_advance')
+	frappe.reload_doc('hr', 'doctype', 'leave_encashment')
+
+	doctype_list = [
+		'additional_salary',
+		'employee_benefit_application',
+		'employee_benefit_claim',
+		'employee_incentive',
+		'employee_tax_exemption_declaration',
+		'employee_tax_exemption_proof_submission',
+		'income_tax_slab',
+		'payroll_entry',
+		'retention_bonus',
+		'salary_structure',
+		'salary_structure_assignment',
+		'salary_slip'
+	]
+
+	for doctype in doctype_list:
+		frappe.reload_doc('payroll', 'doctype', doctype)
+
+	currency_change_list = [
```

Comment:
doctype_list and doctype_list can be commonified using only doctype names ['Leave Encashment'].

you can pass a full doctype name with space in reload_doc.

And in update query you can use like `UPDATE 'tab{0}'`